tests/unit/test_isort.py

```diff
@@ -3287,14 +3290,29 @@ def test_skip_glob(tmpdir, skip_glob_assert: Tuple[List[str], int, Set[str]]) ->
 
     config = Config(skip_glob=skip_glob, directory=str(base_dir))
     skipped = []  # type: List[str]
+    broken = []  # type: List[str]
```

Comment:
This is not a comment on your code, just a realization, we still have type comments! We can probably switch to `broken: List[str] = []` since isort is Python 3.6+ now

Reply:
I thought that there was some reason  that i don't know for using old style typing in testsuite. So you mean it is just legacy, isn't it? :smile: